src/neuron/cache/mechanism_range.hpp

```diff
@@ -22,7 +22,7 @@ void indices_to_cache(short type, Callable callable) {
         auto const sem = dparam_semantics[field];
         // See https://github.com/neuronsimulator/nrn/issues/2312 for discussion of possible
         // extensions to caching.
-        if ((sem > 0 && sem < 1000) || sem == -1 /* area */) {
+        if ((sem > 0 && sem < 1000) || sem == -1 /* area */ || sem == -9 /* diam */) {
```

Comment:
I was reading #2312 and then saw https://github.com/neuronsimulator/nrn/issues/2417. Does this mean `area` and `diam` are already handled and aspects described in #2417 are addressed?

Reply:
Let's close that issue.